src/Driver/solver_configs.jl

```diff
@@ -86,6 +86,7 @@ function SolverConfiguration(
     diffdir = EveryDirection(),
     timeend_dt_adjust = true,
     CFL_direction = EveryDirection(),
+    skip_update_aux = false,
```

Comment:
This is getting pretty ugly. :(

We were looking into removing the update call altogether -- is that really not possible?

Reply:
Agreed. This was the simplest thing I could think of without duplicating the first half of `SolverConfiguration`

Reply:
Wait, why is the `update_aux` call needed here? Did you guys add something that requires it? I'm pretty sure I removed any need for it from the Ocean side

Reply:
The only reason it's needed inside this function is because the atmos courant computations depend on aux variables that depend on the prognostic variables. The init aux method only initializes aux variables that are independent of the prognostic variables-- the update aux method basically initializes the remaining aux variables. The problem is that we cannot use this method (`SolverConfiguration`) without defining prognostic fields, and this unit test should not be concerned with the prognostic field because it's only concerned with aux initialization.

Reply:
I ran into the same issue where I wanted an aux state that depends on the state variables. @jkozdon and I were able to find a workaround that didn't involve changing the DGModel structure too much, but, if there is more than one use case for aux depending on the state, it might make more sense move the initialization from the `DGModel` constructor [1] to the `init_ode_state` function [2]. I'm not sure how invasive this would be; the main issue I can see is if we have initial states that depend on auxiliary variables.

[1] https://github.com/CliMA/ClimateMachine.jl/blob/master/src/Numerics/DGMethods/DGModel.jl#L40-L41
[2] https://github.com/CliMA/ClimateMachine.jl/blob/master/src/Numerics/DGMethods/DGModel.jl#L585-L653

Reply:
This sounds to me like a case for having a separate group of auxiliary variables -- splitting them between those that are independent of the prognostic variables and those that aren't.

Reply:
@jkozdon mentioned they had considered that in the past but never went forward with it

I think that would be a good idea because we definitely have two cases of auxiliary variables: those that are constant in time and those that aren't. It would be nice to split them into two separate concepts.

Reply:
We were initially thinking about it in terms of those that do and do not need to be communicated, but this seems like a reasonable use case as well (the comm issue doesn't matter anymore since @lcw made it so that aux is never communicated after initialization).

Reply:
Going to leave this for now and tackle removing it as a part of closing #1551.